README.md

```diff
@@ -0,0 +1,565 @@
+                        Default is 4.14
+SABAYON_EXTRA_PKGS      Define additional packages to install
+                        on spinbase rootfs.
+SABAYON_UNMASK_PKGS     Define additional packages to unmask.
+SABAYON_ENMAN_REPOS     Define additional enman repository to install
+                        on spinbase rootfs.
+
+```
+
+**NOTE**: Every ISO images are based on spinbase rootfs, so to customize kernel version
+          (that must be available on Sabayon Repository) it is needed build also spinbase
+          ISO image because override of last available kernel is done on inner_script of
+          spinbase image.
+
+*iso_build.sh* script contains a lot of features used by Sabayon Team for release operations,
+features that could be disabled (for example for email through *--skip-email*) and other that can
+be used only but Sabayon Team Group for example: *--push*, *--pushonly* options.
+
+After build is done ISO images are available under *iso* directory.
+
+### Prepare environment
+
+```
+
+// Get molecules sabayon data
+$# git clone https://github.com/Sabayon/molecules.git
+$# cd molecules
+// Became root
+$# sudo su
+
+// Now you can build ISO
+$# SABAYON_MOLECULE_HOME=`pwd` ./scripts/iso_build.sh daily
+```
+
+### Examples
+
+**Build daily spinbase image with kernel 4.9**:
+
+```
+$# cd molecules
+$# SABAYON_KERNEL_VERSION=4.9 SABAYON_MOLECULE_HOME=`pwd` ./scripts/iso_build.sh \
+      daily --image spinbase --logfile /tmp/sabayon-build.log \
+      --stdout --skip-dev --pull-skip --skip-email
+```
+
+Option *--pull-skip* can be used when is already availaboe on local Docker image.
+
+**Build daily Gnome image with kernel 4.9**:
+```
+$# cd molecules
+$# SABAYON_KERNEL_VERSION=4.9 SABAYON_MOLECULE_HOME=`pwd` ./scripts/iso_build.sh \
+      daily --image spinbase --image gnome --logfile /tmp/sabayon-build.log \
+      --stdout --skip-dev --pull-skip --skip-email
+```
+
+**Build daily Gnome image with additional packages**:
+
+```
+$# cd molecules
+$# SABAYON_KERNEL_VERSION=4.9 SABAYON_MOLECULE_HOME=`pwd` \
+      SABAYON_EXTRA_PKGS="app-emulation/lxd games-strategy/wesnoth" \
+      ./scripts/iso_build.sh daily --image spinbase --image gnome \
+      --stdout --skip-dev --pull-skip --skip-email
+
+```
+
+**Build daily Gnome image with additional packages from gaming-live community repository**:
+
+```
+$# cd molecules
+$# SABAYON_KERNEL_VERSION=4.15 SABAYON_MOLECULE_HOME=`pwd` \
+      SABAYON_EXTRA_PKGS="media-libs/mesa-9999 x11-libs/libdrm-9999" \
+      SABAYON_ENMAN_REPOS="gaming-live" \
+      ./scripts/iso_build.sh daily --image spinbase --image gnome \
+        --stdout --skip-dev --pull-skip --skip-email
+
+```
+
+
+## List of specs
+
+Main difference between iso with *dev* extension is related with type of repository used
+for create image: *dev* use limbo repository otherwise sabayon-weekly it is used.
+
+Hereinafter, list of tree of specs file and dependencies:
+
+```
+
+amd64.common
```

Comment:
Hmm I saw this already somewhere, put comment that that detailed information can get out of sync too easily, though it looks very nice.